.github/CODEOWNERS

```
@@ -82,8 +82,8 @@ Tools/patchcheck/             @AA-Turner
 # Autotools
 configure*                       @erlend-aasland @corona10 @AA-Turner @emmatyping
 Makefile.pre.in                  @erlend-aasland @AA-Turner @emmatyping
+Modules/makesetup                @AA-Turner @emmatyping
```

```suggestion
Modules/makesetup                @erlend-aasland @AA-Turner @emmatyping
```
@erlend-aasland Do you also want to be added to this CODEOWNERS entry? (Feel free to apply the suggestion if so).

---

I'd add him out of consistency if nothing else, the files are conceptually related so I'd say it makes sense.

A